vtm-android/build.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'com.github.dcendents.android-maven'
 
 dependencies {
     api project(':vtm')
+    api project(':vtm-mvt')
```

Comment:
Cannot load `vtm-android` module with extra dependencies, which may be not useful by all users.

We could handle it in separate modules, so that everyone can pick only the needed dependencies.